src/Wpf.Ui/Controls/ListView/ListView.cs

```diff
@@ -0,0 +1,60 @@
+namespace Wpf.Ui.Controls;
+
+public class ListView : System.Windows.Controls.ListView
```

Comment:
Missing docs and examples for public control, see: 
https://github.com/lepoco/wpfui/blob/baaf749864df48be57c18821c9dcd47a2b5faa73/src/Wpf.Ui/Controls/Button/Button.cs#L12-L32